project-management-infrastructure/src/main/java/life/qbic/projectmanagement/infrastructure/ontology/TIBTerminologyServiceIntegration.java

```diff
@@ -331,7 +333,16 @@ private Optional<TibTerm> searchByOboIdExact(String oboId)
                 + "&ontology=" + createOntologyFilterQueryParameter()))
         .header("Content-Type", "application/json").GET().build();
     var response = HTTP_CLIENT.send(termSelectQuery, BodyHandlers.ofString());
-    return parseResponse(response).stream().findFirst();
+    log.debug("Received response code '%d' for term query".formatted(response.statusCode()));
```

Comment:
I think it makes sense to include the query term in the log message.
```suggestion
    log.debug("Received response code '%d' for term query %s".formatted(response.statusCode(), oboId));
```